stock_picking_auto_create_lot/i18n/es.po

```diff
@@ -1,44 +1,42 @@
 # Translation of Odoo Server.
 # This file contains the translation of the following modules:
-# 	* stock_picking_auto_create_lot
+#	* stock_picking_auto_create_lot
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: Odoo Server 11.0\n"
+"Project-Id-Version: Odoo Server 12.0\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2018-10-24 20:33+0000\n"
-"PO-Revision-Date: 2018-10-24 22:33+0200\n"
-"Last-Translator: \n"
+"POT-Creation-Date: 2019-02-25 13:29+0000\n"
+"PO-Revision-Date: 2019-02-25 13:29+0000\n"
+"Last-Translator: <>\n"
 "Language-Team: \n"
-"Language: es\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
-"Content-Transfer-Encoding: 8bit\n"
-"Plural-Forms: nplurals=2; plural=(n != 1);\n"
-"X-Generator: Poedit 2.0.6\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
 
 #. module: stock_picking_auto_create_lot
-#: model:ir.model.fields,field_description:stock_picking_auto_create_lot.field_stock_picking_type_auto_create_lot
+#: model:ir.model.fields,field_description:stock_picking_auto_create_lot.field_stock_picking_type__auto_create_lot
 msgid "Auto Create Lot"
-msgstr "Auto crear lote"
+msgstr "Auto Crear Lote"
```

Comment:
You can respect previous translation, or even better, try to unify the source term (with or without upper case in the middle) for not having to translate 2 times the same.

Reply:
> Please sign the CLA

I singned and send icla yesterday

Reply:
OK, then our OCA secretary will insert it and with next commit the acknowledge will appear :)